contracts/Safe.sol

```diff
@@ -164,12 +163,9 @@ contract Safe is
                 gasToken,
                 refundReceiver,
                 // Signature info
-                nonce
+                nonce++
```

Comment:
this is a cleanup from previous EIP-1271 logic change

Reply:
LGTM!